autoload/previm.vim

```diff
@@ -40,6 +40,11 @@ endfunction
 
 function! s:exists_openbrowser() abort
   try
+    if get(g:, 'spacevim_plugin_manager', '') ==# 'dein'
+      if get(dein#get('open-browser.vim'), 'sourced', 0) == 0
+        call dein#source('open-browser.vim')
+      endif
+    endif
```

Comment:
* Why did you add these lines? open-browser.vim is not loaded properly?
* These codes depend on SpaceVim and dein, is it really necessary?